spec/models/aaib_report_spec.rb

```diff
@@ -89,60 +89,53 @@ def aaib_report_content_item(n)
     }
   }
 
-  before do
-    @fields = [
-      :base_path,
-      :content_id,
-    ]
-
-    @aaib_reports = []
+  let(:fields) { %i[base_path content_id] }
 
-    10.times do |n|
-      @aaib_reports << aaib_report_content_item(n)
-    end
+  let(:aaib_reports) { 10.times.map { |n| aaib_report_content_item(n) } }
```

Comment:
I'm inclined to leave this as is. I don't think the more Rubocop way is as clear to people new to Ruby and/or this Application.